BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeHttpClient.kt

```diff
@@ -124,6 +124,7 @@ internal class BraintreeHttpClient(
         if (authorization is TokenizationKey) {
             request.addHeader(CLIENT_KEY_HEADER, authorization.bearer)
         }
+        request.addHeader("Authorization", "Bearer " + authorization?.bearer)
```

Comment:
We don't want to send this with other API requests that hit the Braintree GW. On iOS we did a check like [this](https://github.com/braintree/braintree_ios/blob/235c6994f3c4b5373db25f0f1cd46770e6a38fdd/Sources/BraintreeCore/BTHTTP.swift#L275-L278)

Reply:
Yeah I agree, I just did not see an easy way to do it that wouldn't be messy. I can look more into it.

Reply:
```suggestion
        authorization?.bearer?.let { token -> request.addHeader("Authorization", "Bearer $token")
```
^ is a good option. It guards against `Authorization: Bearer null` from being sent.